src/Components/Body.styled.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
```

Comment:
you dont have installed styled-components on your project. Maybe you installed it globaly? please run "npm install styled-components" and it should add it to your package,json